### DISCUSSION TITLE
Attribution in compiled binaries

### DISCUSSION THREAD

**Discussion by jovermann** [nlohmann/json#3598] [Q&A] [answered]
Do binaries which were compiled using json.hpp need to contain and/or reproduce any attribution to the copyright holders of json.hpp?

If so, in what form does the attribution need to be? E.g. a text file accompanying the binary? Text in the copyright window of a GUI?

My current understanding is that the intention of the MIT license is to keep the license and attribution attached to the source, and that binaries are not affected. But this is under constant debate. What is your take or your wish?

Cheers,
Johannes

**Comment by nlohmann** [comment, marked as answer]
To my understanding (I am not a lawyer), a text in the copyright window of a GUI should be fine. This is how a lot of other companies (like Apple) are doing it, also with MIT licensed software.

**Reply by nlohmann** [reply]
@jovermann Do you need further assistance with this issue?
